### PR TITLE
Serverinfo Plugin

### PR DESCRIPTION
**Pull request by MyNameIsAutistic**
**Plugin Name:** Serverinfo

**Plugin __Short__ Description:** Get info about the current server!

**Your Discord Tag:** Coop#3058

**Your Discord ID:** 710193783576461312

### CODE REVIEW

File: serverinfo_plugin/serverinfo.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+exports.run = async (client, message, args) => {
+const { MessageEmbed } = require('discord.js')
+const server = message.guild
+const e = new MessageEmbed()
+.setColor('RANDOM')
+.setTitle(`Info on guild ${server.name}.`)
+.addField('Server ID',[`${server.id}`],true)
```
Comment (yogurtsyum) [inline serverinfo_plugin/serverinfo.js:7]: Why put the field text in an array?

File: serverinfo_plugin/serverinfo.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+.addField('Members',[`${server.members.cache.size}`],true)
```
Comment (yogurtsyum) [inline serverinfo_plugin/serverinfo.js:8]: Maybe just use server.memberCount instead

File: serverinfo_plugin/serverinfo.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+.addField('Roles',[`${server.roles.cache.size}`],true)
+.addField('Owner ID',[`${server.owner.id}`],true)
```
Comment (yogurtsyum) [inline serverinfo_plugin/serverinfo.js:10]: Maybe ping the owner with `<@${server.owner.id}>`.

**Comment by MyNameIsAutistic** [comment]
Will do. Sorry for the wait.

File: serverinfo_plugin/serverinfo.js (hunk 1 above)
Comment (MyNameIsAutistic) [reply serverinfo_plugin/serverinfo.js:7]: Well, I just saw someone do it one time, then decided I would try it, then i got used to doing it that way. Not even sure how to do it without the array now. Lol.

File: serverinfo_plugin/serverinfo.js (hunk 2 above)
Comment (MyNameIsAutistic) [reply serverinfo_plugin/serverinfo.js:8]: Alright, I will try that.

Comment (MyNameIsAutistic) [reply serverinfo_plugin/serverinfo.js]: Yup, that worked.

File: serverinfo_plugin/serverinfo.js (hunk 3 above)
Comment (MyNameIsAutistic) [reply serverinfo_plugin/serverinfo.js:10]: Alright.

File: serverinfo_plugin/serverinfo.js (hunk 1 above)
Comment (yogurtsyum) [reply serverinfo_plugin/serverinfo.js:7]: Just remove the `[` and `]`.

**Comment by yogurtsyum** [comment]
I would also rename the `README and LICENSE.md` to just `README.md` because then it will show up without someone having to click it.

File: serverinfo_plugin/serverinfo.js (hunk 1 above)
Comment (MyNameIsAutistic) [reply serverinfo_plugin/serverinfo.js:7]: oh ok

**Comment by MyNameIsAutistic** [comment]
> I would also rename the `README AND LICENSE.md` to just `README.md` because then it will show up without someone having to click it.

Alright.

**Pull request merged**